sklearn/metrics/pairwise.py

```diff
@@ -1468,8 +1470,8 @@ def pairwise_distances_chunked(X, Y=None, reduce_func=None,
         reducing it to needed values.  ``reduce_func(D_chunk, start)``
         is called repeatedly, where ``D_chunk`` is a contiguous vertical
         slice of the pairwise distance matrix, starting at row ``start``.
-        It should return an array, a list, or a sparse matrix of length
-        ``D_chunk.shape[0]``, or a tuple of such objects.
+        It should return one of: None; an array, a list, or a sparse matrix
+        of length ``D_chunk.shape[0]``; or a tuple of such objects.
```

Comment:
Update docstring here to state the purpose of a `reduce_func` returning `None` is for inplace operations.

Reply:
Maybe `None when reduction happen inplace` might be a good addition?